ui/raidboss/data/06-ew/raid/p9n.txt

```diff
@@ -1,6 +1,259 @@
+# Kokytos:8127 |   F  | Auto-attack (physical, fighter)
+# Kokytos:8128 |   F  | Archaic Rockbreaker (cast)
+# Kokytos:8129 |   F  | Shockwave
+# Kokytos:812A |   F  | Archaic Rockbreaker (AoE explosion)
+# Kokytos:812B |   F  | Inside Roundhouse
+# Kokytos:812C |   F  | Outside Roundhouse
+# Kokytos:812D |   F  | Swinging Kick (Front Combination)
+# Kokytos:812E |   F  | Swinging Kick (Rear Combination)
+# Kokytos:812F |   F  | Archaic Demolish (cast)
+# Kokytos:8130 |   F  | Archaic Demolish (damage)
+# Kokytos:8131 |   F  | Ascendant Fist
+# Kokytos:8132 |    B | Charybdis (cast, self-targeted)
+# Kokytos:8133 |    B | Charybdis (damage, targets 4 players)
+# Kokytos:8134 |    B | Comet
+# Kokytos:8135 |    B | Comet Impact
+#   Comet:8136 |      | Burst
+# Kokytos:8137 |    B | Touchdown
+# Kokytos:8138 |    B | Beastly Roar
+# Kokytos:8139 |    B | Gluttonous Rampage
+# Kokytos:813A |    B | Ravenous Bite (tankbuster)
+# Kokytos:813B |    B | Ecliptic Meteor
+# Kokytos:813C |    B | Ecliptic Meteor
+# Kokytos:813D |    B | Beastly Bile (cast)
+# Kokytos:813E |    B | Beastly Bile (damage)
+# Kokytos:813F |    B | Pulverizing Pounce
+# Kokytos:8140 |    B | Auto-attack (physical, behemoth)
+# Kokytos:8141 |  M   | Global Spell (cast)
+# Kokytos:8142 |  M   | Global Spell (damage)
+#   Comet:8143 |      | Burst
+# Kokytos:8144 | UM   | ???
+# Kokytos:8146 |    B | ???
+# Kokytos:8147 |   F  | Front Combination (second phase)
+# Kokytos:8148 |   F  | Front Combination (first phase)
+# Kokytos:8149 |   F  | Rear Combination (second phase)
+# Kokytos:814A |   F  | Rear Combination (first phase)
+# Kokytos:814E |  M   | Fire III - Cast
+# Kokytos:814F |  M   | Blizzard III - Cast
+# Kokytos:8188 |    B | Ecliptic Meteor
+# Kokytos:86E1 |  M   | Iceflame Summoning
+#
+# Ice Sphere:86E4 | Sphere Shatter (small)
+# Ice Sphere:86E5 | Sphere Shatter (large)
+# Ice Sphere:872A | Large omen
+# Ice Sphere:8729 | Small omen
+#
+# Fire Sphere:86E2 | Explosion (small)
+# Fire Sphere:86E3 | Explosion (large)
+# Fire Sphere:8727 | Small omen
+# Fire Sphere:8728 | Large omen
+
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+7.0 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/ window 10,10
+10.1 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+18.2 "Gluttony's Augur" sync / 1[56]:[^:]*:Kokytos:8116:/
+18.6 "--sync--" sync / 1[56]:[^:]*:Kokytos:8117:/
+20.3 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+
+
+
+# Mage Phase 1
+26.4 "Ravening (Mage)" sync / 1[56]:[^:]*:Kokytos:8118:/
+35.5 "--sync--" sync / 1[56]:[^:]*:Kokytos:811C:/
+38.6 "--sync--" sync / 1[56]:[^:]*:Kokytos:811C:/
+41.7 "--sync--" sync / 1[56]:[^:]*:Kokytos:8144:/
+48.4 "Fire III" sync / 1[56]:[^:]*:Kokytos:814E:/
+51.7 "--sync--" sync / 1[56]:[^:]*:Kokytos:8122:/
+55.6 "--sync--" sync / 1[56]:[^:]*:Kokytos:811C:/
+56.3 "Fire III" sync / 1[56]:[^:]*:Kokytos:8120:/
```

Comment:
Optional if you want to, but I try to only use real ability names in the timeline, but sometimes there's a need to differentiate.  Here, it could make sense to say `Fire III (cast)` and `Fire III (explode)` or something to differentiate that this is only one `Fire III` and not two.  `golbez-ex.txt` does this a bunch as an example.

Reply:
Ended up with "cast" and "resolve" just so I could match the wording between Fire III and Blizzard III, and I couldn't think of a better word for Blizzard III than "resolve".